modules/planning/tasks/learning_model/learning_model_inference_task.cc

```diff
@@ -73,6 +73,11 @@ Status LearningModelInferenceTask::Process(Frame* frame) {
         absl::StrCat("learning_data adc_trajectory_point empty. frame_num[",
                      learning_data_frame.frame_num(), "]");
     AERROR << msg;
+    // hybrid model will use rule based planning when learning model output is
+    // not ready
+    if (FLAGS_planning_learning_mode == 3) {
```

Comment:
tasks are supposed to be independent of "context", and only takes "parameter" type of gflag(s). so that they can be easily added to any scenario/stage.
tasking context as condition may introduce unexpected behavior, and unnecessary debug effort.

Reply:
Fixed. Thank you.